var/da/da_radiance/module_radiance.f90

```diff
@@ -87,21 +87,41 @@ module module_radiance
 
   ! cf. RTTOV-11 Users Guide Table 3
   ! List of instruments  !!!! HIRS is number 0
-  Character (len=8), Dimension(0:65) :: rttov_inst_name  =             &
```

Comment:
Hi Wei,
I think it might be better if you append new rttov_inst_name behind ‘slstr’ (line 104) and delete lines from 90 to 124.
Junmei

Reply:
The list for rttov_inst_name appears to be in the same order as the instrument list in the RTTOV user guide.  In that case, I think Wei is correct.  It would be good to populate as many instruments of the CRTM list as are available.  There is no need to keep the commented code lines, however.

Reply:
Dear Dr. Ban and JJ, thank you so much for your suggestions. I have updated the code and made another pull request, in which I keep both the same order as the list in the RTTOV and the format of the origin code.